src/or/onion.h

```diff
@@ -121,5 +160,50 @@ int extend_cell_format(uint8_t *command_out, uint16_t *len_out,
 int extended_cell_format(uint8_t *command_out, uint16_t *len_out,
                          uint8_t *payload_out, const extended_cell_t *cell_in);
 
+// TODO: Remove these defines when we implement the first handshake that uses
+// CREATE(D)2V cells. --isis
+#ifndef ONION_HANDSHAKE_TYPE_NTOR2_NULL
```

Comment:
These ifndef wrappers shouldn't be necessary -- instead, only define this constant in one place.